### ISSUE TITLE
Bruno can't Handle 303 Response

### ISSUE THREAD

**Issue by riipandi** [usebruno/bruno#2404]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Looks like Bruno can't handle the response with code 303. I have tried with another API client (I'm using HTTPie) and had no issues.

### Screenshots/Live demo link

This is response on Bruno:
<img width="1912" alt="SCR-20240603-gtgp" src="https://github.com/usebruno/bruno/assets/921834/e77a5463-9d91-4562-83e0-6764260d584b">

This is response on HTTPie Desktop:
<img width="1912" alt="SCR-20240603-gugq" src="https://github.com/usebruno/bruno/assets/921834/e861ec69-1b97-463c-a2ab-01cdc31df399">

**Comment by gemiusz** [comment]
Bruno Version: 1.18.1 and 1.17.0

**Everything working with HTTP 303.**

Looking at error message you have some issue with "access_token".

I try to reproduce and can't.

![image](https://github.com/usebruno/bruno/assets/30750006/eca3dd2d-9d90-4100-97b4-8373a93238a0)


Please provide reproducable code/bru file etc.


My bru file:
```
meta {
  name: HTTP 303
  type: http
  seq: 3
}

get {
  url: https://postman-echo.com/status/303
  body: none
  auth: none
}

```

**Comment by fgrande** [comment]
I can confirm it works also on Bruno 1.19.0

**Comment by riipandi** [comment]
Thanks for the reply for all of you guys, @fgrande @gemiusz. Now, it's already solved by updating to 1.19.0.
